### ISSUE TITLE
Only disallow HTML with equivalent Markdown

### ISSUE THREAD

**Issue by Arcanemagus** [wooorm/remark-lint#32]
Some HTML tags don't have equivalent Markdown and should be allowed to be used, such as `<kbd>` tags.

**Comment by wooorm** [comment]
Very interesting! I definitely think this would be a *great* addition.

However, I’d rather not parse every possible HTML node completely. I mean, It’s possible to pass really funky (and deeply nested) HTML structures as markdown.

So where should we draw the line? I think `<kbd>ESC-u</kbd>` should be allowed by this rule, but should the following from Alex’s readme? (Note the BRs, which are technically possible with Markdown):

```
<h1 align="center">
  <img width="400" src="https://rawgit.com/wooorm/alex/master/media/logo.svg" alt="alex">
  <br>
  <br>
</h1>
```

**Comment by wooorm** [comment]
Fixed my comments: I meant to say that *all* these tags are possible to create with alex.

**Comment by Arcanemagus** [comment]
Honestly I don't , in no way do I consider myself an expert with Markdown, hence why I love tools like remark-lint :stuck_out_tongue:

I just ran into this warning when using `<kbd>`, and figured I would write the issue in a more generic manner since I assumed there were similar tags.

Personally I would think a simple list of allowed tags would be sufficient, maybe configurable in a `.remarkrc` file with the defaults being known tags?

**Comment by wooorm** [comment]
OK, as a first step maybe allow:

*   single opening and closing tags (`<$name>` and `</$name>`) with known HTML names without a markdown counterpart;
*    allow comments.

That should fix most annoying cases.

**Comment by Arcanemagus** [comment]
Sounds like a plan to me.

**Comment by matrixik** [comment]
Another:
- Line break in table cell when using `<br>`

**Comment by wooorm** [comment]
Closing for now, I’m still super open to it, but I don’t have time for it (apparently) on my own backlog. If anyone wants to work on this: ping me!
